linodecli/__init__.py

```diff
@@ -31,6 +31,24 @@
 
 cli = CLI(VERSION, BASE_URL, skip_config=skip_config)
 
+
+def warn_python2_eol():
+    """
+    Prints a warning the first time each day that the CLI is used under python2.
+    This is included to help users upgrade to python3 before python2 support is
+    dropped in the CLI.
+    """
+    if version_info.major < 3:
+        print(
+            "You are running the Linode CLI using python 2, which reached its end of life on "
+            "Jan 1st, 2020.  The Linode CLI will be dropping support for python 2, and "
```

Comment:
```suggestion
            "You are running the Linode CLI using Python 2, which reached its end of life on "
            "Jan 1st, 2020.  The Linode CLI will be dropping support for Python 2, and "
```
When not referring to the executable `python` or `python2`, I believe Python should be capitalized. It is always capitalized in official sources of [python.org](https://python.org).